pbsmetrics/go_metrics.go

```diff
@@ -121,10 +121,13 @@ func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderNa
 		ImpsTypeNative: blankMeter,
 
 		AdapterMetrics: make(map[openrtb_ext.BidderName]*AdapterMetrics, len(exchanges)),
-		accountMetrics: make(map[string]*accountMetrics),
+		accountMetrics: nil,
 
 		exchanges: exchanges,
 	}
+	if !disableAccountMetrics {
```

Comment:
We want to keep account metrics, just disable Account X Adapter metrics. So we need this map in all cases.

Reply:
Rollbacked